server/cache.go

```diff
@@ -525,13 +525,17 @@ func (c *clusterInfo) handleRegionHeartbeat(region *regionInfo) error {
 	}
 
 	// Region meta is updated, update kv and cache.
-	if r.GetVersion() > o.GetVersion() || r.GetConfVer() > o.GetConfVer() {
-		log.Infof("update region %v origin %v", region, origin)
+	if r.GetVersion() > o.GetVersion() {
+		log.Infof("[region %d] %s, Version changed from {%d} to {%d}", region.GetId(), origin.DiffKey(region), o.GetVersion(), r.GetVersion)
```

Comment:
I think print range changed from [start, end) to [start, end) is enough, no need DiffKey here.